Cineme.h

```diff
@@ -0,0 +1,116 @@
+
+#include "Movie.h"
+
+
+class Cinema
+{
+private:
+    vector<Movie> movies;
+    string name;
+    
+    int seats;
+
+public:
+    void printCinema()
+    {
+        cout << "Cinema: " << name << ";" << endl
+             << endl;
+    }
+    Cinema(string tmpName, int tmpSeats)
+    {
+        name = tmpName;
+
+        seats = tmpSeats;
+    }
+
+    void addNewMovieToCinema(Movie movie)
+    {
+        movies.push_back(movie);
+    }
+
+    void filterMovieByGenre(type genreFilter)
+    {
+        for (int i = 0; i < movies.size(); i++)
+        {
+            if (movies.at(i).getGenre() == genreFilter)
+            {
+                movies.at(i).print();
+            }
+        }
+    }
+    
+    void printAllMovies()
+    {
+        for (int i = 0; i < movies.size(); i++)
+        {
+            movies.at(i).print();
+        }
+
+        cout << "Count of movies : " << movies.size() << endl;
+    }
+
+    void sortMoviesByRank()
+    {
+        Movie entry;
+        for (int i = 0; i < movies.size(); i++)
+        {
+            for (int j = 0; j < movies.size() - i - 1; j++)
+            {
+                if (movies.at(j).getRanking() > movies.at(j + 1).getRanking())
+                {
+                    entry = movies.at(j);
+                    movies.at(j) = movies.at(j + 1);
+                    movies.at(j + 1) = entry;
+                }
+            }
+        }
+    }
+};
+
+
+
+tm MakeDate(int day, int month, int year)
+{
+    tm date;
+    date.tm_mday = day;
+    date.tm_mon = month;
+    date.tm_year = year;
+    return date;
+}
+
+class Entry
+{
+public:
```

Comment:
The class members should be wrapped by 4 spaces(or 1 tab).

Reply:
Changed